partcad/tests/unit/test_git_clone_retries.py

```
@@ -83,15 +84,25 @@
 
 test_git_retry_config = {"max": 5, "patience": 0.1}
 
+@pytest.fixture
+def git_config() -> callable:
+    def _git_config(key: str, /, *args, **kwargs) -> dict | str:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)